README.md

```diff
@@ -109,10 +109,10 @@ Participation in the OSCAL Compass community is governed by the [Code of Conduct
 ## License & Authors
 
 If you would like to see the detailed LICENSE click [here](LICENSE).
-Consult [contributors](https://github.com/oscal-compass/compliance-trestle/graphs/contributors) for a list of authors and [maintainers](MAINTAINERS.md) for the core team.
+Consult [contributors](https://github.com/oscal-compass/compliance-trestle/graphs/contributors) for a list of authors and [maintainers](https://github.com/oscal-compass/community/blob/main/MAINTAINERS.md) for the core team.
```

Comment:
I think we should leave this as the `compliance-trestle` maintainers list

Reply:
Yes, of course! The lists need not be the same. I will fix here and the others umbrella projects.